src/CMakeLists.txt

```diff
@@ -154,6 +154,9 @@ include_directories(${glm_INCLUDE_DIRS})
 include_directories(${Vulkan_INCLUDE_DIRS})
 link_directories(${Vulkan_LIBRARY})
 
+find_path(STB_INCLUDE_DIRS NAMES stb_image.h PATH_SUFFIXES stb)
```

Comment:
This feels like it should be in the top level `CMakeLists.txt` with the other `find_library` calls.

Reply:
ok